server.js

```diff
@@ -145,3 +150,38 @@ app.get('/come-back', (req, res) => {
 </body>
 </html>`);
 });
+
+const cspIds = new Map();
+app.get('/security/csp-report/index.html', (req, res) => {
+    const id = crypto.randomInt(2 ** 32).toString(16);
+    cspIds.set(id, []);
+    const policy = `default-src 'self'; img-src 'self'; media-src 'self'; object-src 'none'; script-src 'self' 'unsafe-inline' 'nonce-${id}'; style-src 'self' 'unsafe-inline'; worker-src 'self' blob:; report-uri ${origin}/security/csp-report/csp-report?id=${id}`;
```

Comment:
should we use `req.get('host')` instead of hardcoded `origin` to be more flexible here?